Literate reproducing Taylor coefficients for conformal map

navidcy opened the pull request:
Thought I literate the `compute_Taylor_coefficients.jl` otherwise it gets lost in the sand..

related to #15

Sbozzolo commented:
Minor nitpick, but I find that the progress bar doesn't look great when rendered in a tutorial:
https://clima.github.io/CubedSphere.jl/previews/PR40/literated/compute_taylor_coefficients/

(Also, squash commits before merging)

navidcy commented:
> Minor nitpick, but I find that the progress bar doesn't look great when rendered in a tutorial: https://clima.github.io/CubedSphere.jl/previews/PR40/literated/compute_taylor_coefficients/

True -- I dropped it.

> (Also, squash commits before merging)

Agreed; I changed the defaults of the repo.

navidcy commented:
@Sbozzolo could you have a look?

Sbozzolo commented on examples/compute_taylor_coefficients.jl:
```diff
@@ -16,8 +43,11 @@ function find_angles(φ)
 
-# cbrt goes from W to w
-# cbrt′ goes from W′ to w′
+nothing #hide
```
Why is this needed?

Sbozzolo replied:
(And the other ones below as well)

navidcy replied:
not really needed

Sbozzolo commented on examples/compute_taylor_coefficients.jl:
```diff
@@ -16,8 +43,11 @@ function find_angles(φ)
-# cbrt′ goes from W′ to w′
+nothing #hide
+
+# Next we define the cubic roots and be carefull to choose the appropriate branch.
```
```suggestion
# Next we define the cubic roots and be careful to choose the appropriate branch.
```

Sbozzolo commented:
Preview is here: https://clima.github.io/CubedSphere.jl/previews/PR40/literated/compute_taylor_coefficients/

Looks good to me!
